docs/mysql.md

```diff
@@ -10,7 +10,7 @@ Change the database image to use MySQL instead of PostgreSQL in `compose.yaml`:
 
 ```diff
 ###> doctrine/doctrine-bundle ###
--   image: postgres:${POSTGRES_VERSION:-15}-alpine
+-   image: postgres:${POSTGRES_VERSION:-16}-alpine
```

Comment:
[The latest version](https://github.com/symfony/recipes/blob/8ee3177f13d677e9158cce19479f8b046b8bdfbe/doctrine/doctrine-bundle/2.13/manifest.json#L25) of the recipe uses version `16`